projects/plugins/crm/includes/ZeroBSCRM.PluginAdminNotices.php

```diff
@@ -66,6 +65,11 @@ function jpcrm_woo_promo_admin_notice_banner(){
 add_action( 'admin_notices', 'jpcrm_usage_tracking_notice' );
 function jpcrm_usage_tracking_notice(){
 
+	// remove the notice if white label
```

Comment:
Why not wrap the entire action + function in `##WLREMOVE`?

Reply:
:) can do that for sure, good suggestion